Overview.md

```diff
@@ -0,0 +1,83 @@
+# Overview
+
+Learn a little bit more about Flare's firey APIs
+
+## LocalFirestore

```

Comment:
Remember, local-based Firebase classes are simply mocks of the actual services. While the code examples should use local-based Firebase, please update the rest of the docs to refer to FirebaseFirestore,Auth,Storage.